spec/models/book_spec.rb

```diff
@@ -46,4 +46,12 @@
       expect(result).to eq("default_cover.jpg")
     end
   end
+
+  it 'sets a slug when saved do' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.